playbooks/researchdata.yml

```
@@ -17,5 +19,5 @@
     - name: tell everyone on slack you ran an ansible playbook
       slack:
         token: "{{ vault_pul_slack_token }}"
-        msg: "{{ inventory_hostname }} completed"
+        msg: Ansible ran `{{ ansible_play_name }}` on {{ inventory_hostname }}
```

The quotes aren't necessary here - you only need to quote variables if the phrase/line would otherwise start with `{`.
```suggestion
        msg: Ansible ran {{ ansible_play_name }} on {{ inventory_hostname }}
```